azure-pipelines.yml

```diff
@@ -0,0 +1,31 @@
+trigger:
+- master
+
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: UseGitVersion@5
+  displayName: gitversion
+  inputs:
+    versionSpec: '5.x'
+- task: DotNetCoreCLI@2
+  displayName: dotnet test
+  inputs:
+    command: test
+    arguments: -p:Version=$(GitVersion.MajorMinorPatch)
```

Comment:
Don't think the test should care what the version is :P

Reply:
Oh you're packing it all together, I personally would rather see separate build and then test --no-build
publish --no-build steps than version your test run lol

Reply:
--no-build wasn't working in the test. it took me forever to realize why the package had an asm wih the wrong version

Reply:
so i was doing build, then test with --no-build was actually rebuilding, then pack with no-build was packing the un versioned test. 

im not sure though whether it was that --no-build wasn't working, or that `nobuild: true` desn't work on the task. i suppose I could do custom/test though so long as the option works on `dotnet test`

Reply:
that seems to work